frontend/app-development/features/administration/components/Documentation.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classes from './Documentation.module.css';
+import { Heading, Link, Paragraph } from '@digdir/design-system-react';
+import { ExternalLinkIcon } from '@navikt/aksel-icons';
+import { useTranslation } from 'react-i18next';
+
+export const Documentation = () => {
+  const { t } = useTranslation();
+  return (
+    <div className={classes.documentation}>
+      <Heading level={2} size='xxsmall' className={classes.heading}>
+        {t('administration.documentation.title')}
+      </Heading>
+      <Paragraph size='small' className={classes.content}>
+        {t('administration.documentation.content')}
+      </Paragraph>
+      <Link href='https://docs.altinn.studio/altinn-studio/' className={classes.link}>
```

Comment:
We might consider linking to docs that directly relate to app-development at this stage, f.ex. https://docs.altinn.studio/altinn-studio/designer/build-app/
To get the users to the relevant information even faster 😄